src/py21cmfast/wrapper/outputs.py

```diff
@@ -594,6 +594,16 @@ class OutputStructZ(OutputStruct):
     _meta = True
     redshift: float = attrs.field(converter=float)
 
+    @classmethod
+    def dummy(cls, inputs: InputParameters = InputParameters(random_seed=1)):
+        """Create a dummy instance with the given inputs."""
+        return cls.new(inputs=inputs, redshift=-1.0, dummy=True)
+
+    @classmethod
+    def initial(cls, inputs: InputParameters = InputParameters(random_seed=1)):
+        """Create a dummy instance with the given inputs."""
+        return cls.new(inputs=inputs, redshift=-1.0, initial=True)
+
```

Comment:
I just wanted to double-check that not having the same InputParameters won't mess anything up, either in the parameter checks or the backend. Since all the calls to these constructors are without the input kwarg I'm guessing it doesn't matter, but in that case should we just force the input parameters to be `InputParameters(random_seed=1)` and make sure that they are never called/used?